index.js

```diff
@@ -10,8 +10,8 @@
 
 //. # Sanctuary
 //.
-//. Sanctuary is a small functional programming library inspired by Haskell
-//. and PureScript. It depends on and works nicely with [Ramda][]. Sanctuary
+//. Sanctuary is a functional programming library inspired by Haskell and
+//. PureScript. It depends on and works nicely with [Ramda][]. Sanctuary
 //. makes it possible to write safe code without null checks.
```

Comment:
It's no longer accurate to describe Sanctuary as a *small* functional programming library.